setup.py

```diff
@@ -7,7 +7,14 @@
 with open("CHANGELOG.rst", encoding="UTF-8") as changelog_file:
     history = changelog_file.read()
 
-requirements = ["attrs>=18.1.0", "numpy>=1.11.0", "oop-ext>=1.1", "typing_extensions"]
+requirements = [
+    "alfasim-sdk==0.20.0",
+    "attrs>=18.1.0",
+    "numpy>=1.11.0",
+    "pandas>=2.0.0",
```

Comment:
Do we need pandas?

Reply:
I couldn't put the `pytest_regression` to work without it.

Reply:
mmm